Core/DefInjected/BackstoryDef/Solid_Adult.xml

```diff
@@ -378,11 +378,11 @@
   <CivilEngineer2.titleShort>ingegnere</CivilEngineer2.titleShort>
   
   <!-- EN: [PAWN_nameDef] was bureaucrat in the civil service.\n\nAlthough [PAWN_possessive] family connections took [PAWN_objective] far, [PAWN_possessive] vocal opposition to [PAWN_possessive] bosses' plans hindered [PAWN_possessive] advancement. Called a "red prince" by some, [PAWN_nameDef] sympathized with the poor, leading ultimately to disgrace in [PAWN_possessive] career. -->
-  <CivilServant25.baseDesc>TODO</CivilServant25.baseDesc>
+  <CivilServant25.baseDesc>[PAWN_nameDef] era un burocrate della pubblica amministrazione. Sebbene i legami familiari [PAWN_possessive] abbiano portato [PAWN_objective] lontano, l'opposizione vocale [PAWN_possessive] ai piani dei capi [PAWN_possessive] ha ostacolato l'avanzamento [PAWN_possessive]. Definito da alcuni un "principe rosso", [PAWN_nameDef] simpatizzava con i poveri, portando alla fine alla disgrazia nella carriera di [PAWN_possessive].</CivilServant25.baseDesc>
```

Comment:
Grammatica

Reply:
Ciao, non uso molto github, eventualmente puoi modificare tu i file prima del pull?

Reply:
Yes, no problem. Grazie mille per il tuo prezioso lavoro :)

Reply:
Grazie a voi, considera che nella cartella DefInjected ci sono alcuni file con il TODO ma non c'è la frase in inglese, alcune le ho prese dal tedesco altre no, e forse ho fatto alcuni errori all'interno delle parentesi quadre tipo name con nome.